CubicChunksAPI/src/main/java/io/github/opencubicchunks/cubicchunks/api/util/Coords.java

```diff
@@ -46,7 +46,9 @@
 
     public static BlockPos midPos(BlockPos p1, BlockPos p2) {
         //bitshift instead of / - round always down
-        return new BlockPos((p1.getX() + p2.getX()) >> 1, (p1.getY() + p2.getY()) >> 1, (p1.getZ() + p2.getZ()) >> 1);
```

Comment:
Instead of adding the numbers together and then bitshifting (which risks overflow), we bitshift before we add.

Reply:
he meant an actual java comment, starting with `//`